index.js

```diff
@@ -1 +1,138 @@
-module.exports = require('./lib/emoji');
\ No newline at end of file
+'use strict'
+
+const { default: ow } = require('ow')
+const charRegex = require('char-regex')()
+const emojiRegex = require('emoji-regex')()
+
+const skinTone = require('skin-tone')
+
+const emojiData = Object.entries(require('emojilib').lib).map(([key, { char: emoji }]) => [key, emoji])
+
+const emoji = new Map(emojiData)
+const inverted = new Map(emojiData.map(([key, emoji]) => ([emoji, key])))
+
+function normalizeName (name) {
+  if (/:.+:/.test(name)) {
+    name = name.slice(1, -1)
+  }
+
+  return name
+}
+
+function replace (string, replacement, { removeSpaces = false } = {}) {
+  ow(string, ow.string)
+  ow(replacement, ow.any(ow.string, ow.function))
+  ow(removeSpaces, ow.boolean)
+
+  const replaceFn = typeof replacement === 'function' ? replacement : () => replacement
+
+  const chars = string.match(charRegex)
```

Comment:
```suggestion
  if (chars === null) {
    return string;
  }
```

string.match(...) can return null if there are no matches.